docs/data/date-pickers/fields/DateRangeFieldExamples.tsx

```diff
@@ -24,9 +26,15 @@ export default function DateRangeFieldExamples() {
         <DemoItem label="MultiInputTimeRangeField">
           <MultiInputTimeRangeField defaultValue={[date1, date2]} />
         </DemoItem>
+        <DemoItem label="SingleInputTimeRangeField">
```

Comment:
Could we maybe stick to one order of components in this demo?
Let's go with either of these
- Single -> Multi
- Multi -> Single <- probably makes more sense as this order is used in individual page demos

Reply:
Indeed !
Fixed